src/types.js

```diff
@@ -36,6 +36,26 @@ export function isArray(value) {
 }
 
 /**
+ * Converts an array-like object to an array.
+ * @param {?IArrayLike<*>|string} arrayLike
+ * @return {!Array.<*>}
+ */
+export function toArray(arrayLike) {
+  if (!arrayLike) {
+    return [];
+  }
+  const length = arrayLike.length;
+  if (length > 0) {
```

Comment:
No need for the if condition, since we don't have any legacy code. Everything will be a real array-like, which a numeric `#length`.